### PR TITLE
Add doc for Manage Node Pools of Different VM Types

### LINKED ISSUE #2988: Document node pool management and use of different VM types
## Description

Do we need to add content about Manage Node Pools of Different VM Types

https://kubernetes.slack.com/archives/C02GY94A8KT/p1643719823284929

](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.4/vmware-tanzu-kubernetes-grid-14/GUID-tanzu-k8s-clusters-node-pool.html)

### PR DESCRIPTION
**Pull request by kcoriordan**
## What this PR does / why we need it
Adds documentation for Manage Node Pools of Different VM Types

## Which issue(s) this PR fixes

Fixes: #2988

### CODE REVIEW

File: docs/site/content/docs/node-pool.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+## List Node Pools
+
+To inspect the node pools currently available in a cluster, run:
+
+```shell
+tanzu cluster node-pool list CLUSTER-NAME
+```
+
+This return a lists of all of the node pools in the cluster, and the state of the replicas in each node pool.
```
Comment (stmcginnis) [inline docs/site/content/docs/node-pool.md:29]: s/return/returns/

File: docs/site/content/docs/node-pool.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+## Create a Node Pool
+
+To create a node pool in a cluster:
+
+1. Create a configuration file for the node pool.
+See below for example configuration files for each infrastructure provider.
```
Comment (stmcginnis) [inline docs/site/content/docs/node-pool.md:36]: "See" should be indented to align with "Create" above it.

File: docs/site/content/docs/node-pool.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+  ```yaml
+  name: tkg-aws-wc-np-1
+  replicas: 2
+  az: us-west-2b
+  nodeMachineType: t3.large
+  labels:
+    key1: value1
+    key2: value2
+  ```
```
Comment (stmcginnis) [inline docs/site/content/docs/node-pool.md:70]: Shouldn't be indented.

File: docs/site/content/docs/node-pool.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+  ```yaml
+  name: tkg-azure-wc-np-1
+  replicas: 2
+  az: 2
+  nodeMachineType: Standard_D2s_v3
+  labels:
+    key1: value1
+    key2: value2
+  ```
```
Comment (stmcginnis) [inline docs/site/content/docs/node-pool.md:91]: Shouldn't be indented.

File: docs/site/content/docs/node-pool.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+      numCPUs: 4
+      datacenter: dc0
+      datastore: iscsi-ds-0
+      storagePolicyName: name
+      folder: vmFolder
+      resourcePool: rp-1
+      vcIP: 10.0.0.1
+      template: templateName
+      cloneMode: clone-mode
+      network: network-name
```
Comment (stmcginnis) [inline docs/site/content/docs/node-pool.md:117]: Shouldn't be indented.

File: docs/site/content/docs/node-pool.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+## Delete Node Pools
+
+To delete a node pool run:
+
+  ```shell
+  tanzu cluster node-pool delete <CLUSTER-NAME>
+  ```
+
+   Optionally, use `--namespace` to specify the namespace of the cluster. The default value is `default`.
```
Comment (stmcginnis) [inline docs/site/content/docs/node-pool.md:176]: Shouldn't be indented.

**Review by stmcginnis — COMMENTED** [review]
A few minors things, but this looks good!

**Review by jpmcb — APPROVED** [review]
LGTM - but looks to be a merge conflict @kcoriordan

**Comment by kcoriordan** [comment]
closing this and replacing with:https://github.com/vmware-tanzu/community-edition/pull/4378

**Pull request closed without merging**